packages/coding-components/templates/amd-script-tag/index.html

```diff
@@ -29,17 +29,17 @@
     <script type="module" src="https://js.arcgis.com/calcite-components/1.9.2/calcite.esm.js"></script>
 
     <!-- Load the ArcGIS Maps SDK for JavaScript -->
-    <script src="https://js.arcgis.com/4.28/"></script>
+    <script src="https://js.arcgis.com/next/"></script>
 
     <!-- Load the Coding Components -->
     <link
       rel="stylesheet"
       type="text/css"
-      href="https://js.arcgis.com/coding-components/4.28/arcgis-coding-components.css"
+      href="https://js.arcgis.com/coding-components/next/arcgis-coding-components.css"
```

Comment:
Omar, is components CDN-next is sync with the AMD CDN-next releases? Just double checking.

Reply:
It's in sync on dev. 

js.arcgis.com for components should be in sync again end of February.